packages/shell-api/src/collection.ts

```diff
@@ -96,6 +96,7 @@ import { HIDDEN_COMMANDS } from '@mongosh/history';
 import PlanCache from './plan-cache';
 import ChangeStreamCursor from './change-stream-cursor';
 import { ShellApiErrors } from './error-codes';
+import type * as mql from '@mongodb-js/mql-typescript/schema';
```

Comment:
This is why I'm changing the moduleResolution ts compiler option. I want to be able to import a separate export from the default one.